test/scalability/README.md

```diff
@@ -55,6 +57,8 @@ Setting `SCALABILITY_CPU_PROFILE=1` will generate a cpuprofile of minimalkueue i
 
 Setting `SCALABILITY_KUEUE_LOGS=1` will save the logs of minimalkueue in  `$(PROJECT_DIR)/bin/run-scalability/minimalkueue.out.log` and  `$(PROJECT_DIR)/bin/run-scalability/minimalkueue.err.log`
 
+Setting `SCALABILITY_SCRAPE_INTERVAL` to an interval value (e.g. `1s`) will expose the metrics of `minimalkueue` and have them collected by the scalability runner in `$(PROJECT_DIR)/bin/run-scalability/metricsDump.tgz` every interval. 
```

Comment:
Can we also have a small usability script to load the tgz into victoria metrics?

Reply:
Me neither, but the API seems simple, I successfully loaded a single file with `curl -vX POST -T "$file" http://localhost:8428/api/v1/import/prometheus?timestamp="$timestamp"`. My idea is to use this API to load the dump, similarly as I prototyped in the [doc](https://docs.google.com/document/d/1iO8CGO6kgav2JL6RKYnNYvkeQIHpFqB-SQqJ7eUsjPs/edit?tab=t.0)

Reply:
added a short section on this.